clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -109,15 +109,28 @@ export default async function handler(
 
   const fidesString = environment.settings.FIDES_STRING;
 
-  // Check if a geolocation was provided via headers or query param
-  const geolocation = await lookupGeolocation(req);
+  let geolocation;
+  let propertyId;
 
-  const propertyId = safeLookupPropertyId(
-    req,
-    geolocation,
-    environment,
-    fidesString
-  );
+  try {
+    // Check if a geolocation was provided via headers or query param
+    geolocation = await lookupGeolocation(req);
+
+    propertyId = safeLookupPropertyId(
+      req,
+      geolocation,
+      environment,
+      fidesString
+    );
+  } catch (error) {
+    if (error instanceof Error) {
+      console.error(error);
+      res
+        .status(400) // 400 Bad Request. Malformed request.
+        .send(error.message);
```

Comment:
You can do this instead of checking if `error` is an instance of `Error` and then you could fallback to still return an error message instead of just the empty `return`
```
res
  .status(400) // 400 Bad Request. Malformed request.
  .send(
    error instanceof Error ? error.message : "An unknown error occurred."
  );
```